popit_page.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: PopIt
+siteSlug: popit
+siteName: PopIt
+siteSummary: Create, store and share public lists of politicians
+screenshot: popit.jpg
```

Comment:
Because part of the popit homepage in the screenshot has a white background, there’s this weird ambiguity over where the screenshot ends and the buttons start:

![screen shot 2018-07-11 at 14 26 56](https://user-images.githubusercontent.com/739624/42574571-91cb28e0-8516-11e8-8d79-f231dd41270a.png)

I wonder whether we could bake a light grey border into the image, so it becomes a bit clearer?

Reply:
👍